presto-orc/src/main/java/com/facebook/presto/orc/reader/LongSelectiveStreamReader.java

```diff
@@ -52,9 +52,10 @@ public LongSelectiveStreamReader(
             Optional<Type> outputType,
             OrcAggregatedMemoryContext systemMemoryContext,
             boolean isLowMemory,
-            long maxSliceSize)
+            long maxSliceSize,
+            boolean resetAllReaders)
```

Comment:
Don't introduce resetAllReaders, it does not make much sense.

Reply:
The main purpose of resetAllReaders for performance comparison in prod.
I plan to remove this option if we decide to enable it by default later.
Let me know if this make sense

Reply:
Please remove it now to make it a default behavior.